package.json

```diff
@@ -15,6 +15,7 @@
     "@react-navigation/drawer": "^5.11.4",
     "@react-navigation/native": "^5.8.9",
     "@react-navigation/stack": "^5.12.6",
+    "@types/react-native-vector-icons": "^6.4.6",
```

Comment:
@DiamondRock597 Remove "^" sign, for libraries any minor version of library can take an effect on RN app.